### ISSUE TITLE
Thoughts on the world mender

### ISSUE THREAD

**Issue by xFirefalconx** [immortius/chunkbychunk#67]
Hiya, here's some feedback on the mender. 
On my older machine, I had a significant lag spike while using the Mender on a high amount of chunk spawners. 

- [ ] Hence, a suggestion: How about spawning new chunks based on current performance? This could be to add a custom cooldown on the spawning, or measure the FPS, memory etc. 
- [ ] Observation, there is an interface that shows which chunks were spawned, right. This is rather a huge suggestion, but choosing one chunk and viewing its stats would be nice, or to "re-seal" it again, as you are already considering for other spawner types. A bit like the "Hold [W] to ponder" menus that Create established.
- [ ] I'd love to have a little beginner's manual in book form for my modpack (which is given to each player when they connect to a new world). I have no idea if this is out of scope, but I thought asking won't hurt. Maybe even using something like Patchouli? I love the achievements, but I believe some players might not look at them when trying to figure out how to start.

**Comment by xFirefalconx** [comment]
P.S.: No idea how your LTS plans look like, but I'd love to have such features/improvements for 1.18.2 so I can put them in my Modpack :)

**Comment by immortius** [comment]
I certainly plan to continue to support 1.18.2 for some time yet... it is mostly 1.19.2 and 1.19.3 that I'll probably axe at some point.

It is good to get some feedback on the mender for lower spec machines - I was worried about performance, and while I did try to have a reasonable time gap between spawns if the individual spawn events are proving highly taxing I need to go back and revise my approach for spawning in general. Probably spread it out over more time.

I did try to bake some information into the advancements, but a manual isn't a bad idea. I'll have a look at Patchouli. Not immediately though, I have another project I'll be working on for a bit at least.
